### ISSUE TITLE
Investigate if required transport slots of Vehicles need fixing or adjusting

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#524]
Investigate if required transport slots of Vehicles need fixing or adjusting.

Dozer takes 5 slots
Overlord takes 3 slots
Humvee takes 3 slots
Listening Outpost takes 8 slots

**Comment by commy2** [comment]
Overlord is forbidden from entering Chinooks.
All vehicles are forbidden from entering Infa Assault Helix, but they can still enter Combat Chinooks.

**Comment by ExiLe-ZH** [comment]
Especially ccnooks need to be looked at, can do weird and broken stuff with them.

**Comment by Madrageeeeee** [comment]
yeah you can have 12md's fire out of a ccnook by placing 2 full humvees and 2 MD's into it 

although don't feel this impacts the game too much as it cant really be abused too much i feel. Since the ccnook becomes a seriously high value target at that point. 

being worth 1600 + 3600 + 1200 = 6400 dollar

**Comment by MTKing4** [comment]
An idea just came to me: do you know that lame situation when you have the whole map and well established china army against a camping late game sw general and all he does is sending a bunch of alphas killing all your dozers + CC and hunt you?

I was thinking and actually found a counter to that: just put a dozer in a lix, he'll think you're hunted and suddenly you strike back, hah!

3000 IQ move

Unless...

china Inf: lol fml

Inf lixes should be able to carry vehicles!
(With the exception of outposts and troop crawlers, like all the other chinas)

Imo gives inf more options as well like dragon drops and saving dozer ofc
